### PR TITLE
Biniou+easy format+yojson update

### PR DESCRIPTION
**Pull request by toots**

### CODE REVIEW

**Comment by toots** [comment]
`yojson-windows` failed to build to I updated it.

**Comment by whitequark** [comment]
The format for version update is `easy-format-windows.{1.2.0→1.3.0}`; this allows to see all version updates in git shortlog easily.

File: packages/biniou-windows.1.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+maintainer: "martin@mjambon.com"
+authors: ["Martin Jambon"]
+
+homepage: "https://github.com/mjambon/biniou"
+bug-reports: "https://github.com/mjambon/biniou/issues"
+dev-repo: "https://github.com/mjambon/biniou.git"
+license: "BSD-3-Clause"
+build: [
+  ["jbuilder-windows" "build" "--install-prefix" "windows-sysroot" "-p" "biniou" "-j" jobs "-x" "windows"]
+]
```
Comment (whitequark) [inline packages/biniou-windows.1.2.0/opam]: Missing `remove`

File: packages/easy-format-windows.1.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "martin@mjambon.com"
+authors: ["Martin Jambon"]
+homepage: "http://mjambon.com/easy-format.html"
+bug-reports: "https://github.com/mjambon/easy-format/issues"
+dev-repo: "https://github.com/mjambon/easy-format.git"
+build: [
+  ["jbuilder-windows" "build" "--install-prefix" "windows-sysroot" "-p" "easy-format" "-j" jobs "-x" "windows"]
+]
```
Comment (whitequark) [inline packages/easy-format-windows.1.3.0/opam]: Missing `remove`

File: packages/yojson-windows.1.4.0/opam [hunk 3]
```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "martin@mjambon.com"
+authors: ["Martin Jambon"]
+homepage: "http://mjambon.com/yojson.html"
+bug-reports: "https://github.com/mjambon/yojson/issues"
+dev-repo: "https://github.com/mjambon/yojson.git"
+available: [ ocaml-version >= "4.02.3"]
+build: [
+  ["jbuilder-windows" "build" "--install-prefix" "windows-sysroot" "-p" "yojson" "-j" jobs "-x" "windows"]
+]
```
Comment (whitequark) [inline packages/yojson-windows.1.4.0/opam]: Missing `remove`

**Comment by toots** [comment]
Thanks for the review! The `remove` are implicit, via `<package>-windows.install` files generated by `jbuilder-windows`!

**Pull request merged**